parsons/ngpvan/email.py

```diff
@@ -95,50 +95,48 @@ def get_email_stats(self) -> Table:
             email = self.get_email(fmid)
             email_list.append(email)
 
-        for email in email_list:
-            d = {}
-            d["name"] = email["name"]
-            d["createdBy"] = email["createdBy"]
-            d["dateCreated"] = email["dateCreated"]
-            d["dateModified"] = email["dateModified"]
-            d["dateScheduled"] = email["dateScheduled"]
-            d["foreignMessageId"] = email["foreignMessageId"]
-            d["recipientCount"] = 0
-            d["bounceCount"] = 0
-            d["contributionCount"] = 0
-            d["contributionTotal"] = 0
-            d["formSubmissionCount"] = 0
-            d["linksClickedCount"] = 0
-            d["machineOpenCount"] = 0
-            d["openCount"] = 0
-            d["unsubscribeCount"] = 0
-            try:
+        # Outside and inside emailMessageContentDistributions field
+        outer_fields = [
+            "name",
+            "createdBy",
+            "dateCreated",
+            "dateModified",
+            "dateScheduled",
+            "foreignMessageId",
+        ]
+        inner_fields = [
+            "recipientCount",
+            "bounceCount",
+            "contributionCount",
+            "contributionTotal",
+            "formSubmissionCount",
+            "linksClickedCount",
+            "machineOpenCount",
+            "openCount",
+            "unsubscribeCount",
+        ]
+        if aggregate_ab:
```

Comment:
Can you add some comments to this code, so I can better understand what it's doing? It's hard for me to parse as-is